src/Tools/babylon.tools.ts

```diff
@@ -698,6 +698,12 @@
             return false;
         }
 
+        public static getClassName(obj): string {

```

Comment:
man, if that's still the case I'm in big trouble. I have to test my code on a minified version then and I will keep you updated